benchmarks/buiter_2016_wedge/buiter_2016_wedge.prm

```diff
@@ -0,0 +1,172 @@
+set Maximum time step                      = 1e6
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+#### Parameters describing the model
+
+# Model geometry
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 500
+    set Y repetitions = 100
+    set X extent      = 0.15
+    set Y extent      = 0.035
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left
+  set Prescribed velocity boundary indicators = right:function
+  set Zero velocity boundary indicators = bottom
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = cm=0.01, h=3600., th=0.002
+    set Function expression = if (y>th, -2.5*cm/h, -(y/th)*2.5*cm/h) ; 0.
+  end
+end
+
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top: zero traction 
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set List of model names = box
+
+  subsection Box
+    set Bottom temperature = 100
+    set Left temperature   = 100
+    set Right temperature  = 100
+    set Top temperature    = 100
+
+  end
+end
+
+# Initial temperature field
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 100
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 5
+  set Names of fields  = sand,bound,block,sticky_air,total_strain
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = th=0.002
+    set Function expression = if((x<0.1324 && x>=0.05) && (y<=0.36408*x-0.0182039 && y>=th), 1, 0); \
+                              if((x>=0.1324 && x<0.1329) || (x>0 && y<=th),1,0); \  
+                              if(x>=0.1329 && y>th,1,0); \
+			      if((x<=0.1324 && y>0.36408*x-0.0182039 && y>th) || (x<0.05 && y>th),1,0); \
+                              0;
+  end
+end
+
+# Use disccontinous composition bound preserving limiter
+subsection Discretization
+  set Use discontinuous composition discretization = true
+  subsection Stabilization parameters
+      set Use limiter for discontinuous composition solution = true 
+      set Global composition maximum = 1., 1., 1., 1., 1e5
+      set Global composition minimum = 0., 0., 0., 0., 2e5
+  end
+end
+
+# Material model
```

Comment:
As noted above, somewhere in the documentation you will want to add a full description of how all the values below fit into the model setup. I also suggest adding spaces between "groups" of related values and formatting the equal signs and comma separated values to match up in individual groups. This really helps with readability and you can also add comments above specific groups of parameters.